mycurd/src/main/groovy/pojos/StudentPojo.groovy

```diff
@@ -0,0 +1,15 @@
+package pojos
+
+import com.google.inject.Inject
+import groovy.transform.Canonical
+
+/**
+ * Created by webonise on 24-12-2016.
+ */
+@Canonical
```

Comment:
its good.but start using `@Immutable` everywhere,.there will be compilation errors but that will help you to think about immutable state